src/app/Providers/components/ProvidersTable/ProviderActionsDropdown.tsx

```diff
@@ -93,14 +93,9 @@ const ProviderActionsDropdown: React.FunctionComponent<IProviderActionsDropdownP
         toggleOpen={toggleDeleteModal}
         mutateFn={() => deleteProviderMutation.mutate(provider)}
         mutateResult={deleteProviderMutation}
-        title="Remove provider"
+        title="Remove provider?"
```

Comment:
Let's rephrase this confirmation modal content slightly to align with the other edits, and emphasize that this migration target will transition from selectable to unselectable after users confirm this action:

**Permanently remove provider?**
KubeVirt provider "name" will no longer be selectable as a migration target.